tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -354,6 +355,7 @@ const closeReloadModal = async (shouldUpdate = false, timeout = 5000) => {
   try {
     shouldUpdate ? await modalPage.submit(timeout) : await modalPage.cancel(timeout);
     await modalPage.checkModalHasClosed();
+    shouldUpdate && await waitForAngularLoaded();
```

Comment:
This was what fixed the browser disconnect issue.

Reply:
Nice find! How did you work it out?

I think it would make more sense to have the `await waitForAngularLoaded();` link in the `modalPage.submit` function so nobody else is caught out by this. Unless that breaks other tests that are logging out or something...

Also the above line `await modalPage.checkModalHasClosed();` shouldn't be necessary because the `modalPage.submit` awaits it too.